package.json

```diff
@@ -169,7 +169,6 @@
     "odiff-bin": "2.1.0",
     "parse-github-repo-url": "1.4.1",
     "patch-package": "6.2.2",
-    "percy": "^0.26.9",
```

Comment:
?

Reply:
why is this deleted?

Reply:
percy moved from percy to @percy/cli

Reply:
added cli in prvious PR, forgot to remove this